bool step
float32[] victim_state
---
int32 reward
float32[] state
bool done
